client-src/sass/forms-css.js

```diff
@@ -18,8 +18,8 @@ export const FORM_STYLES = [
       vertical-align: top;
     }
 
-    table td:first-of-type { 
-      width: 60% 
+    table td:first-of-type {
+      width: 60%
```

Comment:
Style: I would prefer a semicolon at the end of the line so that future edits can add or move lines with a cleaner diff and less chance of a mysterious syntax error.

Reply:
This change happened by some auto-formatting feature of vscode, which just removed trailing spaces.  I'll add the semicolons.